pom.xml

```diff
@@ -856,8 +856,8 @@
                                         <mariadb.1011.image>registry.redhat.io/rhel9/mariadb-1011:latest</mariadb.1011.image>
                                         <amqbroker.image>registry.redhat.io/amq7/amq-broker-rhel8:7.12</amqbroker.image>
                                         <!-- Kafka  (amq-streams) requires separate properties for image and version -->
-                                        <amq-streams.image>registry.redhat.io/amq-streams/kafka-35-rhel8</amq-streams.image>
```

Comment:
Lines https://github.com/mjurc/quarkus-test-suite/blob/main/pom.xml#L926-L927 were not updated intentionally?

Reply:
So, I was trying to avoid changing P/Z test containers as I have nowhere to try my changes, but I think I could just update the ones that have multiarch containers for P/Z too, wdyt?